src/index.d.ts

```diff
@@ -55,4 +55,8 @@ export declare function asEither<A, B>(
 export declare function asMaybe<T>(cleaner: Cleaner<T>): Cleaner<T | undefined>
 export declare function asMaybe<T>(cleaner: Cleaner<T>, fallback: T): Cleaner<T>
 
+export declare function asLiteral<
```

Comment:
Change to `asValue`. Remove `{}` type from union, and add `boolean`. Also, change identifier from `literal` to `value`